examples/index.html

```diff
@@ -27,46 +27,49 @@ <h3 id=html>Custom Elements</h3>
 
 <ul>
   <li>
-    <a href=/examples/hello-world>&lt;hello-world&gt;</a>
+    <a title='<hello-world>' href=/examples/hello-world>&lt;hello-world&gt;</a>
 
   <li>
-    <a href=/examples/header-group/example.html>&lt;header-group&gt;</a>
+    <a title='<header-group>' href=/examples/header-group/example.html>&lt;header-group&gt;</a>
 
   <li>
-    <a title=&lt;add-subtract&gt; href=/examples/add-subtract>&lt;add-subtract&gt;</a>
+    <a title='<add-subtract>' href=/examples/add-subtract>&lt;add-subtract&gt;</a>
 
   <li>
-    <a title=&lt;input-output&gt; href=/examples/input-output.html>&lt;input-output&gt;</a>
+    <a title='<input-output>' href=/examples/input-output.html>&lt;input-output&gt;</a>
 
   <li>
-    <a title=&lt;x-flash&gt; href=/examples/x-flashy.html>&lt;x-flashy&gt;</a>
+    <a title='<multiple-choice>' href=/examples/multiple-choice>&lt;multiple-choice&gt;</a>
 
   <li>
-    <a title=&lt;fidget-spinner&gt; href=/examples/fidget-spinner>&lt;fidget-spinner&gt;</a>
+    <a title='<x-flash>' href=/examples/x-flashy.html>&lt;x-flashy&gt;</a>
 
   <li>
-    <a href=/examples/nav-view>&lt;nav-view&gt;</a>
+    <a title='<fidget-spinner>' href=/examples/fidget-spinner>&lt;fidget-spinner&gt;</a>
 
   <li>
-    <a href=/examples/user-list>&lt;user-list&gt;</a>
+    <a title='<nav-view>' href=/examples/nav-view>&lt;nav-view&gt;</a>
 
   <li>
-    <a title=&lt;infinity-calendar&gt; href=/examples/infinity-calendar>&lt;infinity-calendar&gt;</a>
+    <a title='<user-list>' href=/examples/user-list>&lt;user-list&gt;</a>
 
   <li>
-    <a title=&lt;filter&gt; href=/examples/filter.html>&lt;filter&gt;</a>
+    <a title='<infinity-calendar>' href=/examples/infinity-calendar>&lt;infinity-calendar&gt;</a>
 
   <li>
-    <a title=&lt;autocomplete&gt; href=/examples/autocomplete.html>&lt;autocomplete&gt;</a>
+    <a title='<filter>' href=/examples/filter.html>&lt;filter&gt;</a>
 
   <li>
-    <a title=&lt;party-line&gt; href=/examples/party-line.html>&lt;party-line&gt;</a>
+    <a title='<autocomplete>' href=/examples/autocomplete.html>&lt;autocomplete&gt;</a>
 
   <li>
-    <a title=&lt;file-upload&gt; href=/examples/file-upload>&lt;file-upload&gt;</a>
```

Comment:
This diff is a little confusing since it's a rearrange. I might suggest in the future including a seperate PR to for things like rearranging these, since I almost commented that the deletion and addition are different.